F
======================================================================
FAIL: test_failed_example (test_failed_status.TestStatusExample.test_failed_example)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "<string>", line 5, in test_failed_example
AssertionError: my message

----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (failures=1)
